api/_mixins/ParentNode__Document.json

```
@@ -431,7 +431,7 @@
               "version_added": "14"
             },
             "safari_ios": {
-              "version_added": "14"
+              "version_added": "14.5"
```

```suggestion
              "version_added": "14"
```

The original data here seems correct, I can confirm support using https://mdn-bcd-collector.appspot.com/tests/api/Document/replaceChildren in Safari 14.0.1 on iOS 14.2.1 on BrowserStack.

Is there a bug in a script that might have caused this?